azkaban-common/src/main/java/azkaban/executor/Statistics.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.executor;
+
+import java.util.Date;
+
+  public class Statistics {
+    private double remainingMemoryPercent;
+    private long   remainingMemory;
+    private int    remainingFlowCapacity;
+    private Date   lastDispatched;
```

Comment:
I don't have a very strong preference between using Date or long (millis since epoch) to represent time

However, if we are using Date before java 8, I think it's better to use JodaTime.  I think it is a lot more intuitive than the java built in Date functionality.